### DISCUSSION TITLE
com.netflix.graphql.dgs:graphql-dgs-client version compatibility with spring boot 3.x

### DISCUSSION THREAD

**Discussion by vamshers** [Netflix/dgs-framework#1682] [Q&A]
I am currently using com.netflix.graphql.dgs:graphql-dgs-client version 4.9.15. This worked well with spring boot 2.6.6. 
On upgrading  to spring boot 3.x specifically 3.1.3 I am seeing few failures (If required I can add more details about this).

So I upgraded dgs client -  com.netflix.graphql.dgs:graphql-dgs-client  to 7.6.0. After upgrade, few things are working but other things are failing. 

So in general, I want to know if there is a compatibly matrix between dgs framework specifically  **graphql-dgs-client**  and spring boot 3.0.  specifically 3.1.3.  Could some one share any details on the range of dgs clients that are compatible with spring boot 3.x.

**Comment by srinivasankavitha** [comment]
Generally 7.x clients should be compatible with 7.x DGS releases of the framework. Between major versions, we try to avoid any breaking changes, so you should be able to use them.

Hope this heps.
